app/components/job-tabs.js

```diff
@@ -1,11 +1,24 @@
+import { A } from '@ember/array';
+import { computed } from '@ember/object';
 import { isEmpty } from '@ember/utils';
 import Component from '@ember/component';
 
 export default Component.extend({
-
   tagName: 'div',
   classNames: ['travistab'],
 
+  messagesMaxLevel: computed('this.job.build.request.messages.@each.level', function () {
```

Comment:
The path seems to be invalid, context should not be included into it:
```suggestion
  messagesMaxLevel: computed('job.build.request.messages.@each.level', function () {
```